_data/simple-icons.json

```diff
@@ -9848,6 +9848,14 @@
 		"hex": "1B1F24",
 		"source": "https://github.com/mdx-js/mdx/blob/b8a76c95deb14f7297bafdac1aa3eddd2b0fbb8f/docs/_static/icon.svg"
 	},
+	{
+		"title": "Mealie",
+		"hex": "E58325",
+		"source": "https://github.com/mealie-recipes/mealie/blob/mealie-next/frontend/static/icon.png",
+		"license": {
+			"type": "AGPL-3.0-only"
+		}
```

Comment:
The logo/trademark may not licensed under the AGPL-3.0-only. Can you remove this?